ui/src/components/common/SlidingSidebar/partials/VertexDetails/partials/VertexUpdate/index.test.tsx

```diff
@@ -90,15 +90,19 @@ describe("VertexUpdate", () => {
       const mutationBtn = screen.getByTestId("spec-editor-mutated");
       fireEvent.click(mutationBtn);
     });
-    expect(mockSetModalOnClose).toHaveBeenCalledWith(undefined);
+    await waitFor(() => {
```

Comment:
the `act` call for mutationBtn is synchronous, so we do not need the await here. Similarly we can remove the await for `mockSetModalOnClose` below as well.